docs/overview/stability-index.md

```diff
@@ -0,0 +1,470 @@
+
+Stability: `0`
+
+### [newrelic](https://github.com/backstage/backstage/tree/master/plugins/newrelic/)
+
+Observability platform built to help engineers create and monitor their
+software. Stability: `0`
+
+### [proxy-backend](https://github.com/backstage/backstage/tree/master/plugins/proxy-backend/)
+
+A backend plugin used to set up proxying to other endpoints based on static
+configuration.
+
+Stability: `1`
+
+### [register-component](https://github.com/backstage/backstage/tree/master/plugins/register-component/)
+
+A frontend plugin that allows the user to register entity locations in the
+catalog.
+
+Stability: `0`. This plugin is likely to be replaced by a generic entity import
+plugin instead.
+
+### [rollbar](https://github.com/backstage/backstage/tree/master/plugins/rollbar/)
+
+The frontend component of the rollbar plugin, which can be used to view Rollbar
+errors for your services in Backstage.
+
+Stability: `0`
+
+### [rollbar-backend](https://github.com/backstage/backstage/tree/master/plugins/rollbar-backend/)
+
+The backend component of the rollbar plugin, which can be used to view Rollbar
+errors for your services in Backstage.
+
+Stability: `0`
+
+### [scaffolder](https://github.com/backstage/backstage/tree/master/plugins/scaffolder/)
+
+The frontend scaffolder plugin where one can browse templates and initiate
+scaffolding jobs.
+
+Stability: `1`
+
+### [scaffolder-backend](https://github.com/backstage/backstage/tree/master/plugins/scaffolder-backend/)
+
+The backend scaffolder plugin that provides an implementation for templates in
+the catalog.
+
+Stability: `1`. There is planned work to rework the scaffolder in
+https://github.com/backstage/backstage/issues/2771.
+
+### [sentry](https://github.com/backstage/backstage/tree/master/plugins/sentry/)
+
+The frontend component of the sentry plugin, which can be used to view Sentry
+issues in Backstage.
+
+Stability: `0`
+
+### [sentry-backend](https://github.com/backstage/backstage/tree/master/plugins/sentry-backend/)
+
+The backend component of the sentry plugin, which can be used to view Sentry
+issues in Backstage.
+
+Stability: `0`
+
+### [sonarqube](https://github.com/backstage/backstage/tree/master/plugins/sonarqube/)
+
+Components to display code quality metrics from SonarCloud and SonarQube.
+
+Stability: `0`
+
+### [tech-radar](https://github.com/backstage/backstage/tree/master/plugins/tech-radar/)
+
+Visualize the your company's official guidelines of different areas of software
+development.
+
+Stability: `0`
+
+### [techdocs](https://github.com/backstage/backstage/tree/master/plugins/techdocs/)
+
+The frontend component of the TechDocs plugin, used to browser technical
+documentation of entities.
+
+Stability: `1`
+
+### [techdocs-backend](https://github.com/backstage/backstage/tree/master/plugins/techdocs-backend/)
+
+The backend component of the TechDocs plugin, used to transform and serve
+TechDocs.
+
+Stability: `1`
```

Comment:
Are we in a state where we can commit to providing clear upgrade paths when doing major changes to this? I feel the Reader is stable enough, but i'm not sure about the backend. @backstage/techdocs-core

Reply:
Could definitely divide things into sections like some of the other packages too. Feel free to push changes to this branch :grin: